engine/runtime/src/main/scala/org/enso/compiler/pass/resolve/GlobalNames.scala

```diff
@@ -129,23 +129,38 @@ case object UppercaseNames extends IRPass {
               )
             case Right(r @ BindingsMap.ResolvedMethod(mod, method)) =>
               if (isInsideApplication) {
-                lit.updateMetadata(this -->> BindingsMap.Resolution(r))
-
+                lit
+                  .updateMetadata(this -->> BindingsMap.Resolution(r))
+                  .copy(isMethod = true)
```

Comment:
Very strange to modify `isMethod` like this. `isMethod` is a syntactic marker for alias analysis to not try to resolve this name. Shouldn't ever change it.

Reply:
Yeap, leftover from the incremental changes that led to the final solution